cmd/acra_configui/acra_configui.go

```diff
@@ -175,9 +202,9 @@ func main() {
 
 	http.HandleFunc("/index.html", index)
 	http.HandleFunc("/", index)
-	http.Handle("/static/", http.StripPrefix("/static/", http.FileServer(http.Dir("./static"))))
+	http.Handle("/static/", http.StripPrefix("/static/", http.FileServer(http.Dir("./cmd/acra_configui/static"))))
```

Comment:
let's use path to static from cli args/config?

Reply:
Implemented cli-param